config/asset-settings.ts

```diff
@@ -9,7 +9,7 @@ const mainnetAssetSettings: AssetSettings = {
     { key: 'pPool', value: 'PCDAI', type: AssetType.Address },
     { key: 'MaxLoanAmount', value: 25000, type: AssetType.Amount },
     { key: 'MaxTVL', value: 10000000, type: AssetType.Amount },
-    { key: 'MaxDebtRatio', value: 5000, type: AssetType.Uint },
+    { key: 'MaxDebtRatio', value: 10000, type: AssetType.Uint },
```

Comment:
shouldn't we be testing with a more realistic debt ratio?

Reply:
this is from the previous settings we had i think. 

what's a more realistic `MaxDebtRatio`?

Reply:
I'd start with `5000`